src/Symfony/Component/Security/Core/Authorization/Voter/ExpressionVoter.php

```diff
@@ -85,7 +85,7 @@ private function getVariables(TokenInterface $token, $object)
         $variables = array(
             'token' => $token,
             'user' => $token->getUser(),
-            'object' => $object,
+            'object' => $subject,
```

Comment:
Adding `subject` along side `object` in 2.8 is a good idea. Throwing a deprecation notice is possible, but not trivial.